backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -714,34 +764,44 @@ void getPatients_counts() {
     // count includes patients for site2 AND facility=null
-    assertEquals(7, _service.getPatientsCount(site2Id, false, null, false));
+    assertEquals(7, _service.getPatientsCount(site2Id, ArchivedStatus.UNARCHIVED, null, false));
 
     // delete a couple, verify counts
-    List<Person> patients_site2 = _service.getPatients(site2Id, 0, 100, false, null, false);
+    List<Person> patients_site2 =
+        _service.getPatients(site2Id, 0, 100, ArchivedStatus.UNARCHIVED, null, false);
 
     // delete Charles (_site1)
     _service.setIsDeleted(patients_org_page0.get(0).getInternalId(), true);
     // Delete Frank (_site2)
     _service.setIsDeleted(patients_site2.get(0).getInternalId(), true);
 
-    assertEquals(10, _service.getPatientsCount(null, false, null, false));
-    assertEquals(12, _service.getPatientsCount(null, true, null, false));
-    assertEquals(6, _service.getPatientsCount(site2Id, false, null, false));
-    assertEquals(7, _service.getPatientsCount(site2Id, true, null, false));
+    assertEquals(10, _service.getPatientsCount(null, ArchivedStatus.UNARCHIVED, null, false));
+    assertEquals(12, _service.getPatientsCount(null, ArchivedStatus.ALL, null, false));
+    assertEquals(2, _service.getPatientsCount(null, ArchivedStatus.ARCHIVED, null, false));
+    assertEquals(6, _service.getPatientsCount(site2Id, ArchivedStatus.UNARCHIVED, null, false));
+    assertEquals(7, _service.getPatientsCount(site2Id, ArchivedStatus.ALL, null, false));
+    assertEquals(1, _service.getPatientsCount(site2Id, ArchivedStatus.ARCHIVED, null, false));
 
     // counts for name filtering
-    assertEquals(5, _service.getPatientsCount(null, false, "ma", false));
-    assertEquals(2, _service.getPatientsCount(site2Id, false, "ma", false));
-    assertEquals(4, _service.getPatientsCount(site1Id, true, "ma", false));
-    assertEquals(4, _service.getPatientsCount(null, false, "mar", false));
-    assertEquals(2, _service.getPatientsCount(null, false, "MARTHA", false));
-
-    assertEquals(0, _service.getPatientsCount(null, false, "M", false));
-    assertEquals(0, _service.getPatientsCount(null, false, "", false));
+    assertEquals(5, _service.getPatientsCount(null, ArchivedStatus.UNARCHIVED, "ma", false));
+    assertEquals(2, _service.getPatientsCount(site2Id, ArchivedStatus.UNARCHIVED, "ma", false));
+    assertEquals(4, _service.getPatientsCount(site1Id, ArchivedStatus.ALL, "ma", false));
+    assertEquals(4, _service.getPatientsCount(null, ArchivedStatus.UNARCHIVED, "mar", false));
+    assertEquals(2, _service.getPatientsCount(null, ArchivedStatus.UNARCHIVED, "MARTHA", false));
+
+    assertEquals(0, _service.getPatientsCount(null, ArchivedStatus.UNARCHIVED, "M", false));
+    assertEquals(0, _service.getPatientsCount(null, ArchivedStatus.UNARCHIVED, "", false));
+
+    assertEquals(1, _service.getPatientsCount(null, ArchivedStatus.ARCHIVED, "fra", false));
```

Comment:
@fzhao99 -- testing patient count for "ARCHIVED" status here, all facilities or particular facility with text "fra" or "charles"